### ISSUE TITLE
Questions about model structure

### ISSUE THREAD

**Issue by npuichigo** [austinmoehle/wavernn#3]
I used tensorboard to inspect your model structure and found that the pb model you provided just uses one softmax with 256 outputs (8 bits).

![image](https://user-images.githubusercontent.com/11533479/44767932-36564b00-ab92-11e8-8bb5-03ee71288a42.png)

However, the paper uses two separated DNNs to predict the coarse and fine part of a sample. Is that because your model reuse the matrix of O1 and O3 (O2 and O4) or you just support 8 bits with mu-law compression?

![image](https://user-images.githubusercontent.com/11533479/44768138-eb890300-ab92-11e8-97fb-c1079d2fc2e5.png)

**Comment by MlWoo** [comment]
@npuichigo I also have rewritten the training code by the graph.   It does work and the audio sounds good but the waveform is not different from the target.  The systhesised audio is delayed than the target in totally.
